### PR TITLE
Update the documentation of sparse_softmax_cross_entropy_with_logits()

### PR DESCRIPTION
**Pull request by ghost**
With @fd839dc, sparse_softmax_cross_entropy_with_logits() was extended
to allow the type of `labels` to be int32.

Refs #1250

### CODE REVIEW

**Comment by gunan** [comment]
Can one of the admins verify this patch?

**Comment by dtrebbien** [comment]
Is it necessary to regenerate the Python API docs in `tensorflow/g3doc/api_docs/python/`?

**Comment by ebrevdo** [comment]
No; that's not necessary.  It'll be done automatically on a regular basis.

**Comment by ebrevdo** [comment]
@tensorflow-jenkins test this please.
